.github/workflows/tests-unit-php.yml

```diff
@@ -144,19 +144,22 @@ jobs:
 
       - name: Run tests
         run: |
-          npm run test:php
```

Comment:
This file should be renamed to include integration. So tests-unit-integration-php.yml. Also update title of the workflow.

Reply:
Neither would be technically correct as the workflow right now runs both unit and integration tests. Are you perhaps suggesting creating two separate workflows, `tests-php-integration.yml` and `tests-php-unit.yml`?

Reply:
Adding as two different workflow feel a little wasteful, specially as you need to do a composer install and run in different php versions. 

But wonder, if we just rename the file and title.